### ISSUE TITLE
Hamburger menu button briefly shown even if IsFullScreen mode is set True

### ISSUE THREAD

**Issue by toituu** [Windows-XAML/Template10#692]
This is more of a question than bug really but...

Our applications has an landing page for which the hamburger menu is not to be shown. It is only shown after user navigates further from the landing page. It seems that even by setting the IsFullScreen property to True in Shell by default, the menu button is briefly shown on screen before the actual landing page gets loaded.

Is there some trick on how could the hamburger button really be hidden until the menu is to be shown?It's not a big issue, but small annoyance in UX perspective.

**Comment by JerryNixon** [comment]
How can I repeat this?

**Comment by toituu** [comment]
Well, I guess by launching any app using the Template10 Hamburger template. The landing page in this case is very simple page just to query the user credentials. By setting the IsFullScreen to true in the Shell, the hamburger menu button is briefly shown in screen before the actual contents for the landing page is loaded when the app is launched. And I mean brief, just long enough to be seen.
No major logic or other steps involved...

**Comment by toituu** [comment]
Oh, this is mostly issue when testing on phone device. Not as much when testing on desktop. Not too easy to see with emulator. No matter if Debug or Release mode.

**Comment by darshanrampatel** [comment]
@toituu, I'm doing something very similiar to you, but can't replicate this on my 950.

My set-up:
`IsFullScreen="true"` on the `HamburgerMenu` in **Shell.xaml**
`Views.Shell.HamburgerMenu.IsFullScreen = true;` in `OnNavigatedToAsync` in **LoginPageViewModel.cs**

And just before I navigate to my MainPage, I do:

```
Views.Shell.HamburgerMenu.IsFullScreen = false;
NavigationService.Navigate(typeof(Views.MainPage));
```

What does your code look like?

**Comment by JerryNixon** [comment]
I can replicate it. Strange. Just a ghost of the menu.

**Comment by JerryNixon** [comment]
For now, @toituu, I am going to call this an edge case and close this issue. If you come up with a compelling reason why this is wrecking your app, open a new issue and we'll chase it. Just so you know, I tried a few techniques...

Ha! I will leave that part for you to read and enjoy. As I was typing it came to me why this was happening and it was a very simple fix. I have implemented the fix and you will see it in the next NuGet. :) Some of the best ideas come while taking a shower, huh? In this case, the best idea came while trying to describe why I could not come up with the idea. :)
